about/about.css

```diff
@@ -0,0 +1,83 @@
+*{
+    margin:0;
+    padding: 0;
+}
+.background-image{
+    background-size: cover; 
+    height:500px; 
+    padding-top:80px;
+    text-align:left;
+}
+.box {
+    background: url(images/codecomplogo.jpg);
+    padding: 100px;
+    margin: 100px;
+}
+h1 {
+    font-size:70px; 
+    color:hsl(225, 100%, 23%); 
+    margin:10px;
+}
+
+body {
+    font-family: Georgia, Arial, Helvetica;
+    margin:0;
+}
+html {
+    box-sizing: inherit;
+}
+*, *:before, *:after {
```

Comment:
Two colons should be used for CSS [pseudo-element](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements) selectors.